### PR TITLE
Modernize gem/CI

### PR DESCRIPTION
**Pull request by n-rodriguez**
Thank you!

### CODE REVIEW

**Comment by n-rodriguez** [comment]
ping @radar thank you!

**Review by fz-ryanbigg — APPROVED** [review]

**Comment by radar** [comment]
@tubbo I can't merge this one without your approval. Can you please take a look?

**Comment by n-rodriguez** [comment]
ping @tubbo

File: Gemfile [hunk 1]
```diff
@@ -1,6 +1,11 @@
+# frozen_string_literal: true
+
 source 'https://rubygems.org'
 
 gemspec
 
+gem 'connection_pool', '~> 1.2.0'
```
Comment (mperham) [inline Gemfile:7]: This is quite old.

Comment (n-rodriguez) [reply Gemfile]: Right. Thank you!

**Review by tubbo — APPROVED** [review]

**Comment by tubbo** [comment]
sorry about the holdup folks!

**Comment by n-rodriguez** [comment]
> sorry about the holdup folks!

No worries, thank you! 👍

**Pull request merged**